control-plane/CMakeLists.txt

```diff
@@ -96,19 +96,17 @@ set (CONTROLPLANE_HDRS
 )
 
 add_library(controlplane-gen OBJECT ${P4RUNTIME_GEN_SRCS})
-# TODO(https://github.com/p4lang/p4c/issues/4477):
-# We should not need this, but because of the way managed includes and
-# libraries for Protobuf we need to explicitly add the include directories here.
-target_include_directories(
-  controlplane-gen SYSTEM BEFORE PUBLIC ${Protobuf_INCLUDE_DIRS} SYSTEM BEFORE
-  PUBLIC ${ABSL_COMMON_INCLUDE_DIRS}
+
+target_include_directories(controlplane-gen
+  # TODO(https://github.com/p4lang/p4c/issues/4477):
+  # We should not need this, but because of the way managed includes and
+  # libraries for Protobuf we need to explicitly add the include directories here.
+  SYSTEM BEFORE PUBLIC ${Protobuf_INCLUDE_DIRS}
```

Comment:
While we are here, maybe solve protobuf include madness? Can we just link protobuf as library?

Reply:
Since we pushed the minimum Protobuf version to 3.25 we should be able to clean this up yeah. Was planning on doing it once I have some more cycles.